### PR TITLE
update clojure, go, php, ruby and scala buildpacks

### PR DESCRIPTION
**Pull request by vtrmantovani**
Update buildpacks
clojure -> 78
go ->85
php -> 131
ruby ->174
scala-> 79

### CODE REVIEW

File: rootfs/builder/install-buildpacks [hunk 1]
```diff
@@ -29,14 +29,14 @@ download_buildpack() {
...
-download_buildpack https://github.com/heroku/heroku-buildpack-clojure.git        v77      02-clojure
-download_buildpack https://github.com/heroku/heroku-buildpack-go.git             v81      03-go
+download_buildpack https://github.com/heroku/heroku-buildpack-clojure.git        v78      02-clojure
+download_buildpack https://github.com/heroku/heroku-buildpack-go.git             v85      03-go
 download_buildpack https://github.com/heroku/heroku-buildpack-gradle.git         v23      04-gradle
 download_buildpack https://github.com/heroku/heroku-buildpack-grails.git         v21      05-grails
 download_buildpack https://github.com/heroku/heroku-buildpack-java.git           v59      06-java
 download_buildpack https://github.com/heroku/heroku-buildpack-nodejs.git         v118     07-nodejs
-download_buildpack https://github.com/heroku/heroku-buildpack-php.git            v121     08-php
+download_buildpack https://github.com/heroku/heroku-buildpack-php.git            v131     08-php
```
Comment (yagonobre) [inline rootfs/builder/install-buildpacks:38]: This jump 10 versions and change many lib versions, if anyone is using PHP, it would be nice to test this.

**Review by yagonobre — APPROVED** [review]
LGTM

**Review by drgarcia1986 — APPROVED** [review]
great, thank you!

**Review by aguerra — APPROVED** [review]
LGTM

**Pull request merged**